packages/synapse-react-client/src/mocks/msw/handlers/wikiHandlers.ts

```diff
@@ -1,34 +1,133 @@
+import {
+  FileHandleResults,
+  ObjectType,
+  WikiPage,
+  WikiPageKey,
+} from '@sage-bionetworks/synapse-types'
 import { rest } from 'msw'
-import { ACCESS_REQUIREMENT_WIKI_PAGE } from '../../../utils/APIConstants'
-import { FileHandleResults, WikiPage } from '@sage-bionetworks/synapse-types'
+import { WIKI_OBJECT_TYPE, WIKI_PAGE } from '../../../utils/APIConstants'
+import { MOCK_WIKI_ETAG, NEW_WIKI_PAGE_ID, mockWikiPages } from '../../mockWiki'
+import { mockWikiPageKeys } from '../../mockWikiPageKey'
+import { MOCK_USER_ID } from '../../user/mock_user_profile'
 import { SynapseApiResponse } from '../handlers'
-import { mockWikiPages } from '../../mockWiki'
 
-export const getWikiHandlers = (backendOrigin: string) => [
```

Comment:
Updated to also handle calls for other wiki owner object types (ENTITY, EVALUATION)